ci-operator/step-registry/cucushift/installer/check/ibmcloud/custom-endpoints/cucushift-installer-check-ibmcloud-custom-endpoints-ref.yaml

```diff
@@ -42,3 +42,23 @@ ref:
     default: ""
     documentation: |-
       GlobalTagging endpoint
+  - name: SERVICE_ENDPOINT_COSConfig
+    default: ""
+    documentation: |-
+      COSConfig endpoint
+  - name: SERVICE_ENDPOINT_GlobalCatalog
+    default: ""
+    documentation: |-
+      GlobalCatalog endpoint
+  - name: SERVICE_ENDPOINT_KeyProtect
+    default: ""
+    documentation: |-
+      KeyProtect endpoint
+  - name: SERVICE_ENDPOINT_CIS
+    default: ""
+    documentation: |-
+      CIS endpoint, which is not used in disconnect cluster.
+  - name: SERVICE_ENDPOINT_HyperProtect
```

Comment:
I did not see the default endpoint for HyperProtect defined in eps_default.json, if we are not ready to test it, pls remove it.

Reply:
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58984/rehearse-58984-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.17-installer-rehearse-ibmcloud-dis/1862457795660484608 

> url: Invalid value: "https://api.private.us-east.hs-crypto.cloud.ibm.com/": Head "https://api.private.us-east.hs-crypto.cloud.ibm.com/": Service Unavailable

Based on dev's replay, this maybe cost reason. How about keep it, just not set "DEFAULT_ENDPOINT" in _cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private-provision-chain.yaml_